README.md

```diff
@@ -141,6 +141,18 @@ Decide how many nodes you want to run.
 
 TODO
 
+### Exposing elasticsearch service with a route
+
+Obtain the CA cert from Elasticsearch.
+```
+oc extract secret/elasticsearch --to=. --keys=admin-ca
+```
+Create a Secured Route Using Re-encrypt Termination.
+(See also https://docs.openshift.com/container-platform/3.11/architecture/networking/routes.html#secured-routes)
+
+In the Re-encrypt termination example, put the CA cert from Elasticsearch to spec.tls.destinationCACertificate.  
+The rest spec.tls.key, spec.tls.certificate and spec.tls.caCertificate are no need to be set.
```

Comment:
s/The rest spec.tls.key, spec.tls.certificate and spec.tls.caCertificate are no need to be set./You do not need to set the spec.tls.key, spec.tls.certificate and spec.tls.caCertificate parameters shown in the example.

Reply:
Thanks!  Updating it...